esp_wrover_kit/esp_wrover_kit.c

```diff
@@ -202,7 +205,7 @@ static lv_disp_t *bsp_display_lcd_init(void)
     esp_lcd_panel_io_handle_t io_handle = NULL;
     esp_lcd_panel_handle_t panel_handle = NULL;
     const bsp_display_config_t bsp_disp_cfg = {
-        .max_transfer_sz = BSP_LCD_DRAW_BUFF_SIZE * sizeof(uint16_t),
+        .max_transfer_sz = 10 * BSP_LCD_H_RES,
```

Comment:
not sure how much performance we will improve if we increase this size. Should we have a small test on it and choose a prpoer one?  10 lines seems is the recommendation for LVGL draw buffer